DMTN-049.tex

```diff
@@ -229,85 +230,131 @@ \section{Technical Considerations from the DM System}\label{sec:dmcon}
+\textbf{LOR Purpose:}
+The Rubin science community has a considerable wealth of expertise in generating PZ catalogs and will be the primary users of the Object PZ data products.
+These letters provide a formal opportunity for the science community to define their minimum scientific needs regarding the LSST Object PZ data product. 
 
-Based on the above goals, these "Letters of Recommendation":
-\begin{itemize}
-\item should address the technical considerations in \S~\ref{sec:dmcon}
-\item should address the evaluation criteria in  \S~\ref{sec:eval}
-\item should be short (i.e., 1-3 pages, ideally), and should not constitute an onerous task
-\item could focus on many, a few, or just one estimator(s) or science goal(s)
-\item could be primarily {\it qualitative} in their evaluation of the photo-$z$ estimators
-\item could include or reference {\it quantitative} analyses of simulated LSST data, or data from other facilities
-\item will be openly shared in the Rubin community
-\end{itemize}
+\textbf{LOR Writers:}
+Any group or individual who would use the LSST Object PZ for their future scientific analyses, and/or are developers of potentially suitable PZ estimators, are encouraged to submit an letter.
+The DM team is especially interested in hearing the needs of scientists who plan to use the LSST Object catalog but who would/could not generate custom PZ estimates.
 
-It is understood that "Letters of Recommendation" from members of the science community who are future users of the {\tt Object} catalog photo-$z$, but who are not developers of photo-$z$ estimators, would focus on recommending third-party software based on their science needs, and might not be able to fully address the technical considerations.
+\textbf{LOR Scope:}
+Letters should recommend minimum attributes of the LSST PZ data product that would enable basic LSST science, describe the science that the LSST PZ data product should enable, and/or qualitatively recommend a certain PZ estimator (or type of estimator).
+Letters do not have to recommend a certain PZ estimator, and can just focus on describing PZ-related science needs.
+Quantitative analyses are beyond the scope of these letters, as are discussions on the research and development of new or improved algorithms.
 
-However, "Letters of Recommendation" from teams who also plan to support the validation and implementation of one or more estimators during the commissioning-era Photo-$z$ Validation Cooperative (and beyond, to operations) should also address the following technical considerations as best they can (summarized from Section \ref{sec:dmcon}):
+\textbf{Beyond the LORs:}
+The Rubin DM team will use these letters to inform the evaluation criteria used to select PZ estimators, and to assemble a shortlist of viable PZ estimators to be evaluated quantitatively using Rubin Commissioning data during the ``PZ Validation Cooperative" phase of the PZ Roadmap.
```

Comment:
Above you say that LORs do not have to recommend a specific PZ estimator, and here that DM will use the PORs to assemble a shortlist of estimators. Will the shortlist be comprised of PZ estimators recommended in the LORs, or are you saying that DM will build its own list based on the evaluation criteria?  (Note I know you are trying to keep the barriers to entry low so that people will not feel they can contribute without recommending a specific estimator, or have done deep quantitative analysis).

Reply:
I absolutely wanted DM to be able to add estimators to the shortlist if they seem like they'd be viable yet no LOR mentioned them (especially since its hard to tell what kind of participation level we'll get). So I added:
"DM might add viable PZ estimators to the shortlist if DM thinks that they meet the evaluation criteria and the community's scientific needs, even if they were not mentioned by any of the LORs."